runtime_lib/airhost/include/airbin.h

```diff
@@ -0,0 +1,20 @@
+#ifndef AIRBIN_H
+#define AIRBIN_H
+
+#include <cstdint>
+#include <fstream>
+
+uint64_t airbin2mem(std::ifstream &infile, volatile uint32_t *tds_va,
+                    uint32_t *tds_pa, volatile uint32_t *data_va,
+                    uint32_t *data_pa, uint8_t col);
+
+struct airbin_size {
+  uint8_t start_col = 0;
+  uint8_t num_cols = 1;
+  uint8_t start_row = 1;
+  uint8_t num_rows = 2;
+};
+
+airbin_size readairbinsize(std::ifstream &infile, uint8_t column_offset);
+void readairbin(std::ifstream &infile);
```

Comment:
How does this relate to `air_load_airbin`? It should be a C API if it gets `#included` in `air_host.h` which is supposed to be a C header*. The alternative is to `#include` it in `air.hpp` which is much less desirable if these functions are required to use airbin.

\*  and `<string>` has already crept in